src/transaction/log_replication.cpp

```diff
@@ -280,15 +280,21 @@ namespace cublog
   {
     // skip calculation if bogus input (sometimes, it is -1);
     // TODO: fix bogus input at the source if at all possible (debugging revealed that
-    // it happens for LOG_COMMIT messages only)
+    // it happens for LOG_COMMIT messages only and there is no point at the source where the 'at_time'
+    // is not filled in; the only other possibility is a reading misalignment that remains below the radar)

```

Comment:
I think it's best to keep unconfirmed theories out of the code :P
```suggestion
    // it happens for LOG_COMMIT messages only and there is no point at the source where the 'at_time'
    // is not filled in
```

Reply:
Done